book-service/src/main/resources/application-local.properties

```diff
@@ -0,0 +1,22 @@
+springfox.documentation.swagger.v2.path=/api-docs
+server.port=8080
+spring.jackson.date-format=com.anitechcs.bookservice.RFC3339DateFormat
+spring.jackson.serialization.WRITE_DATES_AS_TIMESTAMPS=false
+logging.level.io.swagger.models.parameters.AbstractSerializableParameter=error
+
+#H2 db configuration
+spring.datasource.url=jdbc:h2:mem:testdb
```

Comment:
Can we use the same DB name as Postgres?

Reply:
done

Reply:
do i need to rebase with master or will be done automatically?